core/templates/pages/exploration-player-page/services/current-interaction.service.ts

```diff
@@ -71,7 +71,8 @@ export class CurrentInteractionService {
   }
 
   registerCurrentInteraction(
-      submitAnswerFn: SubmitAnswerFn, validityCheckFn: ValidityCheckFn): void {
+      submitAnswerFn: SubmitAnswerFn | null,
```

Comment:
ditto why would this be `null`?

Reply:
The interaction can pass in null if it does not use the progress nav's submit button. Added the comments as well.